frontend/src/components/Analysis/AnalysisTable/index.tsx

```diff
@@ -267,9 +274,7 @@ export function AnalysisTable({ systemID, task, cases, page, setPage }: Props) {
         pageSize,
         // conversion between 0-based and 1-based index
         current: page + 1,
-        onChange: (newPage, newPageSize) => {
-          setPage(newPage - 1);
-        },
+        onChange: (newPage) => onPageChange(newPage - 1),
```

Comment:
In the antd [docs](https://ant.design/components/pagination/), `onChange` has two parameters `newPage` and `newPageSize`. Is this a special syntax to ignore the second parameter, or should it be put back?

Reply:
This is a somewhat "quirky" thing about JS/TS. You don't need to write the second parameter here if you don't use it in your function. It will be ignored automatically.

Reply:
I see!

Reply:
Actually, this is the wrong way to explain it. Basically in JS, if you have
```
function func(a) {return a;}
```
you can call it with `func(1)`, `func(1,2)` and `func(1,2,3,4,5,6,7)` and it won't complain.